### PR TITLE
Cases in target labels

### PR DESCRIPTION
**Pull request by wilkus27**
Issue: https://github.com/openstax/kitchen/issues/333

### CODE REVIEW

**Review by ajluk — COMMENTED** [review]
I'm only leaving a comment about not working link to documentation. Natalia a lot of work has been done here within this PR! :tada:

**Review by ajluk — APPROVED** [review]

File: lib/locales/pl.yml [hunk 1]
```diff
@@ -1,14 +1,49 @@
...
+    locative: Rysunku
+    vocative: Rysunku
+  table:
+    nominative: Tabela
+    genitive: Tabeli
+    dative: Tabeli
+    accusative: Tabelę
+    instrumental: Tabelą
+    locative: Tabeli
+    vocative: Tabelo
```
Comment (LostMojo) [inline lib/locales/pl.yml:17]: Does Polish use all the cases? Otherwise, looks good

**Review by LostMojo — APPROVED** [review]

File: lib/locales/pl.yml (hunk 1 above)
Comment (LostMojo) [reply lib/locales/pl.yml:17]: *the Polish book, the language, sorry

Comment (wilkus27) [reply lib/locales/pl.yml:17]: Unfortunately yes :smile: Polish language is complicated.

File: lib/kitchen/element_base.rb [hunk 2]
```diff
@@ -67,6 +67,9 @@ class ElementBase
     # @!method remove_attribute
     #   Removes an attribute from the element
     #   @see https://www.rubydoc.info/github/sparklemotion/nokogiri/Nokogiri/XML/Node#remove_attribute-instance_method Nokogiri::XML::Node#remove_attribute
+    # @!method key?(attribute)
+    #   Returns true if attribute is set
+    #   @see https://www.rubydoc.info/github/sparklemotion/nokogiri/Nokogiri%2FXML%2FNode:key%3F Nokogiri::XML::Node#key?(attribute)
```
[the remark this answers is not in the document]
Comment (wilkus27) [reply lib/kitchen/element_base.rb]: Thank you! It's fixed now :slightly_smiling_face:

**Pull request merged**